client/src/Observable.ts

```diff
@@ -65,7 +65,11 @@ export default class Observable {
             let callback;
             if (type === Observable.WILDCARD) {
                 callback = function(this: Observable) {
-                    this.fire.apply(this, arguments);
+                    const args = arguments;
+                    if (typeof args === 'object' && args && typeof args[0] === 'string') {
+                        const argsArray = [ ...arguments ];
+                        this.fire.apply(this, [argsArray[0], ...argsArray.slice(1)]);
+                    }
```

Comment:
Would something like `this.fire.apply(this, arguments as [string, ...any]);` have been possible?
(I don't know whether typescript even supports something like that though).

Otherwise `this.fire.apply(this, [arguments[0] as string, ...[...arguments].slice(1)]);`?

Reply:
```
this.fire.apply(this, [...arguments] as [string, ...any[]]);
```
works.